### PR TITLE
uaasscrum-1340 agencyflow on umbracocloud

### PR DESCRIPTION
**Pull request by mikkelhm**

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Organizations 
+On Umbraco Cloud it is possible to be part of an organization. And organization is a way for you to associate users to one organization.
+The benefits are currently limited, but we are working on improving the features available for organizations.
+Currently organizations are created by Umbraco HQ - so **reach out to support** to hear more about the feature, and weather they apply for your scenario.
+
+## Invite organization to collaborate
+Once you have an organization you will be able to invite other organizations to collaborate for you.
+By inviting an organization the following is will happen
```
Comment (NicolaiOksen) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md]: Type here:
By inviting an organization the following **is** will happen
Should be:
By inviting an organization the following will happen

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* You allow a superadmin in that organization to create projects on behalf of your organization
+* You will be able to choose from super admins in the invited organization, when adding members to a project.
+  * This effectly enalbes the super admin in that organization access to your project, but also allows her to invite other member from that organization.
```
Comment (NicolaiOksen) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md]: Typos here:
effectly --> effectively
enalbes --> enables
access to --> to access
member --> members

Fixed line:
 This effectively enables the super admin in that organization to access your project, but also allows her to invite other members from that organization.

**Comment by sofietoft** [comment]
This is great, guys! 👏 

Could we perhaps add some steps for each of the gifs as well? Or would that be too much?

I just don't like that we have two gifs right on top of each other. Would look better if there was some text between them! I have some ideas. I'll see if I can get around to it today 😁

**Comment by sofietoft** [comment]
@NicolaiOksen , @mikkelhm ..
I've made some *minor* adjustments.

Could you give it a quick look?

**Comment by NicolaiOksen** [comment]
@sofietoft looks good to me :)

**Comment by NicolaiOksen** [comment]
Since it was only typos I fixed, I'm merging :D

**Comment by hemraker** [comment]
This documentation does not make sense unless you already know what a Cloud organization is. I have created an issue for more explanatory docs regarding organizations here #718

**Pull request merged**